src/components/Dashboard.tsx

```diff
@@ -156,7 +156,7 @@ const PanelComponent: React.FC<{
     <>
       {filters.map((filter) =>
         totalMatchFilter[filter.id] > 0 ? (
-          <Link to={constructLink(filter)}>{`${filter.title} (${
+          <Link key={filter.id} to={constructLink(filter)}>{`${filter.title} (${
```

Comment:
Unrelated, but fixes a long-standing key error

Reply:
That's funny my linting/typescript analyzer isn't giving this error - I wonder if my set up isn't the same as yours

Reply:
It's a runtime error (more of a warning really) in the console about a missing key (All iterative components require a unique key so React can keep track of which is which in subsequent re-renders, as the order might change)